### ISSUE TITLE
The site's footer is inaccessible through tabbing because of the unbounded infinite scroll on the main feed

### ISSUE THREAD

**Issue by patricio-ezequiel-hondagneu-roig** [forem/forem#11020]
I chose dev.to as my preferred site to perform manual accessibility testing for a certification, and found that in the front page, I'm effectively unable to reach the footer by tabbing, because once I reach near the bottom of the main feed, it automatically loads more posts.

This is an accessibility violation, as described by Deque Systems in this article: [Infinite Scrolling & Role=Feed Accessibility Issues](https://www.deque.com/blog/infinite-scrolling-rolefeed-accessibility-issues/).

Infinite scroll in the front page's article feed renders the footer inaccessible for keyboard users.

1. Visit the front page
2. Tab into the article feed
3. Try to tab your way to the footer

**Expected behavior**

Eventually you reach the footer

**Actual behavior**

The infinite scroll logic keeps adding articles below the focused element every time I advance

**Suggestion**

Once the user reaches the end of the feed, show a "Load more" button that has to be manually clicked instead of automatically adding articles to the feed.

**Comment by citizen428** [comment]
Thanks for the issue @patricio-ezequiel-hondagneu-roig. I'll close this as a duplicate of https://github.com/forem/forem/issues/9034, please feel free to add to the discussion there.
